script.js

```diff
@@ -12,61 +12,16 @@ function checkFlexGap() {
   document.body.appendChild(flex);
   var isSupported = flex.scrollHeight === 1;
   flex.parentNode.removeChild(flex);
-  console.log(isSupported);
 
   if (!isSupported) document.body.classList.add('no-flexbox-gap');
 }
 checkFlexGap();
 
-// https://unpkg.com/smoothscroll-polyfill@0.4.4/dist/smoothscroll.min.js
+function getFullYear() {
+  const yearEl = document.querySelector('.copyright-year');
+  const currentYear = new Date().getFullYear();
 
-/*
-.no-flexbox-gap .main-nav-list li:not(:last-child) {
-  margin-right: 4.8rem;
+  return (yearEl.innerHTML = currentYear);
```

Comment:
Here you don't need to return a value, it's enough if you set `yearEl.innerHTML = currentYear`